mockgcp/mockgkehub/gkehubfeature.go

```diff
@@ -87,8 +87,10 @@ func (s *GKEHubFeature) UpdateFeature(ctx context.Context, req *pb.UpdateFeature
 		switch path {
 		case "labels":
 			obj.Labels = req.Resource.GetLabels()
-		case "multiclusteringress":
+		case "spec":
```

Comment:
Wow - that's a very broad update!  I think we need to replace the whole spec here?